Week1/homework/app.js

```diff
@@ -3,9 +3,173 @@
 {
   const bookTitles = [
     // Replace with your own book titles
-    'harry_potter_chamber_secrets'
+    'harry_potter_chamber_secrets',
+    'lotr_return_the_king',
+    'yeraltindan_notlar',
+    'simyaci',
+    'insan_ne_ile_yasar',
+    'dogunun_limanlari',
+    'semerkand',
+    'ince_memed',
+    'kurk_mantolu_madonna',
+    'dava',
+    'donusum'
   ];
+  console.log(bookTitles);
+
+  const books = [
+    {
+      id: 0,
+      title: 'harry_potter_chamber_secrets',
+      language: 'english',
+      author: 'j_k_rowling',
```

Comment:
Why did you use underscores and lowercase in the author name? This serves no purpose, and was not asked for in the assignment. Just have a string in there with the name of the author exactly as you want to display it.